Block/Adminhtml/Support/Edit/Tab/ConfigurationSettings.php

```diff
@@ -166,10 +166,10 @@ protected function _prepareForm()
             ]
         );
         $fieldset->addField(
-            'configValues',
+            'orderHistoryComments',
             'textarea',
             [
-                'name' => 'configValues',
+                'name' => 'orderHistoryComments',
                 'label' => __('Attach config values'),
```

Comment:
The label says "Attach config values", but the field name is "orderHistoryComments". 
```suggestion
                'name' => 'orderHistoryComments',
                'label' => __('Order History Comments'),
```